git-workflow.md

```diff
@@ -0,0 +1,101 @@
+# Git Workflow
+
+## Stable Branches
+- `master` - used for production. Must stay stable at all time. Contains the release history.
+- `develop` - used for staging. Contains the development edge.
+- `fastlane` - used for fastlane. Contains the development edge for an epic.
+
+## Development branches
+All development work should be done in a dedicated branch. This 
+makes it easy for multiple developers to work on a particular feature without 
+disturbing the main codebase. It also means the `master` branch will never 
+contain build braking code. 
+
+## Workflow
+### Development
+When ready to start development branch out from `master`
+
+```
+git checkout master
+git pull
+git checkout -b feature/#1234-strong-passwords
+```
+
+```
+master  *-*-*
+             \
+feature/#n    \-*-*-*-*
+```
+
+### Code Review
+Once you have finished a feature, open a PR to merge into `develop`.
+
+### Acceptance
+The development branch is merged into `develop`, which should trigger a deployment
+on CircleCI.
+
+```
+master      ----*
+                 \
+feature/#n        \-*-*-*-*
+                           \
+develop     ----------------m
+```
+
+### Production
+Deployments to production are completed by creating a release branch from master and
+successively merging the selected\* feature branches into the release branch. This allows features
+to be 'cherry-picked' into production without having to wait for all features in develop
+to receive Product Owner approval. 
+
+\**Waffle is currently used to to mark issues by adding them to the "Deploying" column. Both relsr and flightplan use the
+waffle labels to identify issues for the next release.*
+
+```
+master          ---------*-------------------m-------
+                   \      \                 /
+release/20171231-101500    \----m-------m--*
+                     \         /       /
+feature/#n            \-*-*-*-*       / 
+                       \       \     / 
+feature/#x              \-*-*-*-\-*-* 
+                         \       \   \ 
+bug/#b                    \-*-*   \   \ 
+                               \   \   \ 
+develop     --------------------m---m---m----------
+```
+
+To create the release branch and PR you can either use [relsr](https://github.com/jcleary/relsr) on the command-line or allow [flightplan](https://flightplan.createl.io) to create
+them for you on the deployment schedule (currently PR at 10:15, merge at 10:30 daily).
+
+### Resolving Conflicts
+If GutHub warns you of conflicts in your PR, resolve the conflicts by merging your branch into develop on the command line. 
+Do you use the GutHub tools or the command line to merge develop into your branch as this will create deployment dependencies. 
```

Comment:
I think you meant to say 'Do not use GitHub tools or the command line...'. There's another 'GutHub' at the start of the paragraph. 

I can only imagine what a 'GutHub' might look like... haha